### PR TITLE
Added isRequired validaiton on multiSelect

### PR DESCRIPTION
**Pull request by oratiu**

### CODE REVIEW

File: apps/crn-frontend/src/network/working-groups/__tests__/WorkingGroupOutput.test.tsx [hunk 1]
```diff
@@ -94,8 +94,6 @@ describe('WorkingGroupOutput', () => {
     userEvent.click(screen.getByText('Abu-Remaileh, M 1'));
     userEvent.click(screen.getByRole('textbox', { name: /Labs/i }));
     userEvent.click(screen.getByText('Example 1 Lab'));
-    userEvent.click(screen.getByRole('textbox', { name: /Authors/i }));
-    userEvent.click(screen.getByText('Person A 3'));
```
Comment (kmaid) [inline apps/crn-frontend/src/network/working-groups/__tests__/WorkingGroupOutput.test.tsx:98]: ?

File: packages/react-components/src/organisms/AuthorSelect.tsx [hunk 2]
```diff
@@ -57,6 +57,8 @@ const AuthorSelect: React.FC<AuthorSelectProps> = (props) => (
   <LabeledMultiSelect<AuthorOption>
     {...props}
     creatable={true}
+    isRequired={props.isRequired}
```
Comment (kmaid) [inline packages/react-components/src/organisms/AuthorSelect.tsx:60]:
```suggestion
```
its already there from ...props.

File: packages/react-components/src/atoms/MultiSelect.tsx [hunk 3]
```diff
@@ -122,6 +124,8 @@ export type MultiSelectProps<T extends MultiSelectOptionsType> = {
   readonly values?: OptionsType<T>;
   readonly sortable?: boolean;
   readonly creatable?: boolean;
+  readonly isRequired?: boolean;
```
Comment (kmaid) [inline packages/react-components/src/atoms/MultiSelect.tsx:127]: Every other field has this as "required" please refactor this to be consistent.

File: apps/crn-frontend/src/network/working-groups/__tests__/WorkingGroupOutput.test.tsx (hunk 1 above)
Comment (oratiu) [reply apps/crn-frontend/src/network/working-groups/__tests__/WorkingGroupOutput.test.tsx:98]: I moved it to the mandatory fields as authors are mandatory

File: packages/react-components/src/atoms/MultiSelect.tsx [hunk 4]
```diff
@@ -159,6 +164,32 @@ const MultiSelect = <T extends MultiSelectOptionsType>({
...
+
+  const initialRender = useRef(true);
+  const checkValidation = useCallback(() => {
+    if (initialRender.current) {
+      initialRender.current = false;
+    } else {
+      validate();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (kmaid) [inline packages/react-components/src/atoms/MultiSelect.tsx:181]: Why isn't initialRender in this list? Shouldn't it be called inputRef?

File: packages/react-components/src/atoms/MultiSelect.tsx [hunk 5]
```diff
@@ -234,7 +264,17 @@ const MultiSelect = <T extends MultiSelectOptionsType>({
...
-      <div css={validationMessageStyles}>{validationMsg}</div>
+      <input
+        {...validationTargetProps}
+        tabIndex={-1}
+        autoComplete="off"
+        value={values.map((value) => value.label).join('')}
+        required={isRequired}
+        disabled={!enabled}
+        hidden={true}
+        onChange={noop}
```
Comment (kmaid) [inline packages/react-components/src/atoms/MultiSelect.tsx:275]: Why is this set to noop? Can this be omitted or set to undefined?

File: packages/react-components/src/templates/ResearchOutputTeamForm.tsx [hunk 6]
```diff
@@ -321,6 +321,7 @@ const ResearchOutputPage: React.FC<ResearchOutputPageProps> = ({
                 onChangeTeams={setTeams}
                 getTeamSuggestions={getTeamSuggestions}
                 isEditMode={isEditMode}
+                authorsRequired={publishingEntity === 'Working Group'}
```
Comment (kmaid) [inline packages/react-components/src/templates/ResearchOutputTeamForm.tsx:324]: Why is there a condition here? Won't the Team Output form only deal with Team Outputs?

File: packages/react-components/src/atoms/MultiSelect.tsx (hunk 4 above)
Comment (oratiu) [reply packages/react-components/src/atoms/MultiSelect.tsx:181]: Following the approach we have in Dropdown.tsx, we don't want to validate on the first render so we don't display the error message directly.

File: packages/react-components/src/organisms/ResearchOutputContributorsCard.tsx [hunk 7]
```diff
@@ -49,6 +50,7 @@ const ResearchOutputContributorsCard: React.FC<
   isSaving,
   isEditMode,
   authorsSubtitle = '(optional)',
+  authorsRequired = false,
```
Comment (kmaid) [inline packages/react-components/src/organisms/ResearchOutputContributorsCard.tsx:53]:
```suggestion
  authorsRequired
```

File: packages/react-components/src/templates/ResearchOutputTeamForm.tsx (hunk 6 above)
Comment (oratiu) [reply packages/react-components/src/templates/ResearchOutputTeamForm.tsx:324]: We can either temporarily send a prop from the WorkingGroupOutputs component and have that prop added in the TeamForm, which will have to be removed once we add the WorkingGroupForm, or have this in place, which will also have to be removed once we add the WorkingGroupForm.

**Pull request merged**